draft-west-cookie-incrementalism.md

```diff
@@ -224,6 +233,59 @@ attribute (Section 4.1.2.5 of {{RFC6265bis}}) by altering the storage model defi
 This is conceptually similar to the requirements put into place for the `__Secure-` prefix (Section
 4.1.3.1 of {{RFC6265bis}}).
 
+## Schemeful Same-Site {#schemeful-samesite}
```

Comment:
Optional, since it seems I'm the only one who hates "schemeful" :-)

"Redefining Same-Site"?

Reply:
I'll pass on this suggestion for now :)